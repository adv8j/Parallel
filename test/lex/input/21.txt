@parallel(reduction=[+:sum])
for c in 0..100 {
    sum += someArray(a[d]);
}
