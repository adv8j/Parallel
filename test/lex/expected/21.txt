PARALLEL
LPAREN
REDUCTION
ASSIGN
LBRACKET
PLUS
COLON
IDENTIFIER
RBRACKET
RPAREN
FOR
IDENTIFIER
IN
INT_LITERAL
RANGE
INT_LITERAL
LBRACE
IDENTIFIER
ADD_ASSIGN
IDENTIFIER
LPAREN
IDENTIFIER
LBRACKET
IDENTIFIER
RBRACKET
RPAREN
SEMICOLON
RBRACE
